mne/preprocessing/ica.py

```diff
@@ -2180,14 +2180,18 @@ def _plot_corrmap(data, subjs, indices, ch_type, ica, label, show, outlines,
     """Customized ica.plot_components for corrmap"""
     import matplotlib.pyplot as plt
 
-    title = 'Detected components of type ' + label
+    title = 'Detected components'
+    if label is not None:
+         title += ' of type ' + label
+
     picks = list(range(len(data)))
 
     p = 20
     if len(picks) > p:  # plot components by sets of 20
         n_components = len(picks)
         figs = [_plot_corrmap(data[k:k + p], subjs[k:k + p],
-                indices[k:k + p], ch_type, ica, label)
+                indices[k:k + p], ch_type, ica, label,
+                outlines, cmap, contours, layout, show)
```

Comment:
for avoding bugs we should pass the args by keywords. I'm sometimes too lazy to follow this advice but in general we should do this.